### DISCUSSION TITLE
git package not installing latest relased curl

### DISCUSSION THREAD

**Discussion by esmith55** [chocolatey-community/chocolatey-packages#2526] [Q&A]
using the C:\ProgramData\chocolatey\choco.exe upgrade git.install -y --params "/GitAndUnixToolsOnPath /NoGitLfs /SChannel /NoAutoCrlf" to upgrade from 2.45 to 2.46.  The install of git is successful yet the latest version of curl.exe has not been installed but rather version 8.9.0.  is there a way to only update the "curl.exe" or is there way to run the upgrade command and force the "curl.exe" upgrade?

**Comment by rimrul** [comment]
No. Git for Windows 2.46.0 comes with curl 8.9, even if other curl versions exist. There is no installer option to get a different curl version from somewhere.
